src/main/java/io/cdap/plugin/gcp/bigquery/relational/SQLExpressionFactory.java

```diff
@@ -51,6 +52,7 @@ public Expression compile(String expression) {
   @Override
   public Set<Capability> getCapabilities() {
     return new HashSet<>(Arrays.asList(StringExpressionFactoryType.SQL,
+                                       StandardSQLCapabilities.BIGQUERY,
```

Comment:
nit: given this grows up, it makes sense to make it a constant

Reply:
Please update javadoc, it does not reflect the implementation anymore

Reply:
Updated